packages/block-library/src/gallery/style.scss

```diff
@@ -1,8 +1,10 @@
-.wp-block-gallery {
+.wp-block-gallery,
```

Comment:
In the case where Gallery block has a caption then it produces HTML where `.blocks-gallery-grid` is enclosed in `.wp-block-gallery`, won't it cause any issues?

Reply:
I've tested this and added an override on the editor side for `figure.wp-block-gallery` to `display:block` as the flex display was causing issues. It seems to be fine on the post rendering side so far, but I can always add that override globally if needed. The other styles are pretty harmless.